back/engines/commercial/report_builder/Rakefile

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
```

Comment:
Also created by the generator. Indeed, not very useful in this case since our engines are typically too dependent on the host application. Will remove.